### ISSUE TITLE
A11y_Model Builder_Create New Resource group_Info and relationships: Narrator focus is not shifting to 'Create new resource group' popup when it is opened and Narrator announces as 'operation timed out'.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1594]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11y_Model Builder; #A11yMAS; #A11yTCS; #WCAG1.3.1; #Info and Relationships; #Narrator; #Win32; #DesktopApp; #A11ySev2; #ModelBuilder-Win32-Jun2021; #Visual Studio Client;

### Environment Details:
Application name: Model Builder

### OS Details:
Windows Version 21H1 OS Build (19043.1083)
Microsoft Visual Studio

### Note:
This issue is not observed when NVDA is turned on. NVDA is reading the content inside 'Create new resource group' popup.

### Repro Steps:

1. Turn on Narrator. Enable ML.NET Model builder as a preview feature in tools>options>Environment>preview features.
2. Create a .NET C# project and tab till project under solution explorer and Add machine learning.
3. Tab till Image classification under scenario tab and press enter.
4. Select Azure under environment tab and tab till set up workspace and press enter.
5. In 'Create New Experiment' dialog, tab till 'New' link beside Machine learning workspace name field and press enter
6. Tab till 'New' link beside Resource group link and press enter
7. Verify whether Narrator focus is shifting inside 'Create new resource group' popup when it is opened

### Actual Result:
Narrator focus is not shifting to 'Create new resource group' popup when it is opened and Narrator announces as 'operation timed out'.

### Expected Result:
Narrator focus should go to the revealed content i.e.,'Create new resource group' popup when it is opened and Narrator should read all the content wherever focus is landing on. Narrator should not announce as operation timed out when popup is opened.

### User Impact:
Screen reader users will face difficulty in knowing the content if Narrator is not announcing about the popup.

**Comment by zewditu** [comment]
After accessibility team office hour there is a follow-up email with Scott O'Hara to answer offline.

**Comment by zewditu** [comment]
In Teams call discussion with Scott O'Hara , this might be a narrator issue and we have a follow-up email with the narrator team.
Scott also recommend making a design change to completed Azure workspace set up  with less stack of windows

**Comment by beccamc** [comment]
@zewditu Does this work with NVDA? If it does, can we get a video of it working and move this to validate?

**Comment by zewditu** [comment]
yes it works
 https://images.zenhubusercontent.com/186917455/0602216b-66b4-46c2-b530-44a11df0e5a5/nvdaok.mp4

**Comment by beccamc** [comment]
This is a timeout problem with Windows Narrator. We have reached out to the narrator team to investigate their bug, but this is not a problem with our application. To validate please check NVDA, where the correct behavior happens (see video attached by Zewditu above)

**Comment by NiharikaTipirishetty** [comment]
@beccamc Could you please provide sample app where similar issue exists to raise an external bug against Narrator. 

Note: This issue is not observed with NVDA

**Comment by beccamc** [comment]
@NiharikaTipirishetty I will work on getting the sample app. Can the priority of this bug be lowered since it is a Narrator problem?

**Comment by beccamc** [comment]
I've created a small app, but I can't get a repro of this problem. I can't reproduce this problem in Model Builder either. See the following video. All the dialogs are reading correctly in narrator: 

https://user-images.githubusercontent.com/9122518/135354864-2cec0d9b-c898-49a8-9367-27472d3bcfe3.mp4

**Comment by beccamc** [comment]
Here is the latest Model Builder version https://devdiv.visualstudio.com/_apis/resources/Containers/8811121/VSSetup_Release17?itemPath=VSSetup_Release17%2FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix

**Comment by NiharikaTipirishetty** [comment]
@beccamc Could you please confirm if this feature is publicly available? If the feature is publicly available we don't need sample app. Once the bug is logged against Narrator we will be closing this bug as it is working fine with NVDA.

**Comment by NiharikaTipirishetty** [comment]
Closing this bug as this issue is not observed in latest model builder version. Narrator is reading all the info in 'Create new resource group' popup.

![1594_no issue](https://user-images.githubusercontent.com/83570638/135404259-4c853aa6-b4c8-412d-b92e-48258720d66d.png)

**Comment by VijayaBharathyA** [comment]
#Closed;
